terragrunt/aws/network/vpc.tf

```diff
@@ -0,0 +1,50 @@
+#
+# VPC
+#
+
+module "url_shortener_vpc" {
+  source            = "github.com/cds-snc/terraform-modules?ref=v0.0.31//vpc"
+  name              = var.product_name
+  billing_tag_value = var.billing_code
+  high_availability = true
+  enable_flow_log   = true
+  block_ssh         = true
+  block_rdp         = true
+
```

Comment:
[Could you this same logic to the VPC](https://github.com/cds-snc/scan-files/blob/233cca8d3f400d35616d76b71479ddf0238d7554/terragrunt/aws/api/vpc.tf#L10).  It'll help save us money since we'll create less NAT Gateways in Staging, which are billed per hour.

It means Staging won't be truly highly available since if the one NAT Gateway goes down, the API will no longer be able to connect out to the internet, but this is totally fine for non-Prod.

Reply:
Done!